src/dispatch/static/dispatch/src/tag_type/NewEditSheet.vue

```diff
@@ -126,6 +126,21 @@
                   <span> If activated, only one tag of this type is allowed per incident. </span>
                 </v-tooltip>
               </v-col>
+              <v-col cols="5">
+                <v-checkbox
+                  v-model="required"
+                  label="Required"
+                  hint="Is this tag type required for new entities of the discoverable types?"
+                />
+              </v-col>
+              <v-col cols="7">
+                <v-tooltip max-width="500px" open-delay="50" location="bottom">
+                  <template #activator="{ props }">
+                    <v-icon class="mt-4" v-bind="props">mdi-information</v-icon>
+                  </template>
+                  <span> If activated, only one tag of this type is allowed per incident. </span>
```

Comment:
Is it "only one" or is it "at least one"?

Reply:
For exclusive, it is only one.